R/lohView.R

```diff
@@ -36,10 +38,10 @@
 #' @export
 
 lohView <- function(x=NULL, path=NULL, fileExt=NULL, y=NULL, genome='hg19',
-                    step=500000, window_size=1000000, normal=50,
+                    gender=FALSE, step=500000, window_size=1000000, normal=50,
```

Comment:
`gender=FALSE` should be changed to `NULL` for readability. We should change this PR to not calculate X or Y chromosomes unless a vector of length equal to the number of samples is provided. This vector should consist of elements from the set {'M', 'F'}.